src/Jenssegers/Mongodb/Query/Builder.php

```diff
@@ -600,7 +617,12 @@ public function insert(array $values)
      */
     public function insertGetId(array $values, $sequence = null)
     {
-        $result = $this->collection->insertOne($values);
+        $options = [];
```

Comment:
Common code for many methods, better move this logic in special method used in many places